python/jax/carfac.py

```diff
@@ -2352,6 +2354,7 @@ def run_segment(
 
   n_ch = hypers.ears[0].car.n_ch
   naps = jnp.zeros((n_samp, n_ch, n_ears))  # allocate space for result
+  naps_fibers = jnp.zeros((n_samp, n_ch, 3, n_ears))
```

Comment:
Is this "3" the 3 from SynDesignParameters.n_classes ? I think it is, right? 

If so, is it possible to reference that 3 by name instead of the constant here.

Reply:
done